src/pages/book/publish.js

```diff
@@ -68,7 +73,7 @@ class publish extends Component {
     );
 
     return (
-      <Grid className={classes.adjustTop} container>
+      <Grid container className={classes.root}>
```

Comment:
className={classes.root} is unnecessary.

Grid is a wrapper for flexbox. Thus when you use it, you already have the flex defined! to make it column, simply specify direction="column".

see more here :https://material-ui.com/api/grid/

Reply:
Thanks. I will  fix it.